umm-lib/src/cmr/umm/echo10/granule.clj

```diff
@@ -181,6 +200,19 @@
   (when-let [value (util/extract-between-strings xml "<RestrictionFlag>" "</RestrictionFlag>" false)]
     (Double/parseDouble value)))
 
+(comment
+
+  (cmr.umm.echo10.echo10-core/umm->echo10-xml pge-gran)
+
+  
+                        ;;(def my-pvc pge-version-class)
+                      ;;(x/element :PGEVersionClass {} pge-version-class)
+
+  ;;my-pvc from :keys below
+  ;; #cmr.umm.umm_granule.PGEVersionClass {:pge-name "Ak|", :pge-version "iC2&t"}
```

Comment:
If it's in a comment block they can be uncommented but should be functional. Otherwise this could be removed.